help/destinations/catalog/marketing-automation/rainfocus.md

```diff
@@ -0,0 +1,157 @@
+---
+title: RainFocus
+description: Learn how to use the RainFocus Destination Connector to syncronize audience profiles with the RainFocus Global Attendee Profile .
+---
+
+# RainFocus {#rainfocus-destination}
+
+## Overview {#overview}
+
+The RainFocus destination connector allows organizations to stream customer profiles from the Adobe Experience Platform into the RainFocus platform in order to create/update Attendee Profiles. 
```

Comment:
In Experience Platform, we adopt a user-centric approach to documentation, centering our writing around what user can do with certain functionality. In this case, I would suggest rephrasing as:

Use the RainFocus destination to stream customer profiles from Adobe Experience Platform into the RainFocus platform in order to create and update attendee profiles.